clib/bitv_string.mli

```diff
@@ -0,0 +1,76 @@
+(**************************************************************************)
+(*                                                                        *)
+(*  Copyright (C) Jean-Christophe Filliatre                               *)
```

Comment:
This header should also be in the .ml

Reply:
You're right. Thanks for pointing this out. Now fixed in the `bitv` repo.